lib/fhmongodb.js

```diff
@@ -153,13 +153,19 @@ Database.prototype.tearUp = function (auth) {
           if (interval) {
             clearInterval(interval);
           }
-          self.db = db.db(self.name);
```

Comment:
@philbrookes  No I haven't but will now. If it doesn't work its no problem, I will emit the events on the initial db object and then proceed as before. Will let you know if it passes